CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   `upsert`, `upsert_object`, `upsert_many`, `upsert_object_many`,
   `update`, `delete`. (#267).
 
+### Fixed
```

Comment:
`bugfix:` in a commit message is not a module subsystem. Use `schema:` or something like this instead